src/index.html

```diff
@@ -1,6 +1,28 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="use-credentials"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="use-credentials"
+    />
```

Comment:
There are duplicate preconnect links for 'https://fonts.googleapis.com' and 'https://fonts.gstatic.com'. It's redundant to have multiple preconnect links for the same URL, as it can lead to unnecessary network requests. Consider removing the duplicates to optimize performance.